vuejs/src/store.js

```diff
@@ -17,6 +18,24 @@ export default new Vuex.Store({
     },
   },
   getters: {
+    getGender(state) {
+      return state.userInfo.gender === 'male' ? '男性' : '女性';
+    },
+    getBirthday(state) {
+      return `${state.userInfo.year}年${state.userInfo.month}月${state.userInfo.date}日`;
```

Comment:
`{}`の両内側にはスペースを空けましょう。
```suggestion
      return `${ state.userInfo.year }年${ state.userInfo.month }月${ state.userInfo.date }日`;
```